resources/views/home.blade.php

```diff
@@ -93,7 +92,6 @@
                     <div class="form-group">
                         {{ Form::label('name', trans('dashboard.label.add_widget')) }}
                         <select class="form-control" id="add_widget_id" name="add_widget_id" data-dashboard_id="{{ $request->route('dashboard_id') }}">
-                                <option value=""></option>
```

Comment:
I've left this in as if you want to add two of the same widget you would have to refresh the page.